test/NewsService.Data.UnitTests/NewsRepositoryTests.cs

```diff
@@ -90,34 +101,39 @@ public void CleanDb()
 
         #region EditNews
         [Test]
-        public void ShouldThrowExceptionWhenNewsForEditDoesNotExist()
+        public void Success()
```

Comment:
So strange name, mb you should separate it for few tests and name it more specifically?

Reply:
I am testing the same kind of query but with different combinations of properties. For any of the following requests, the operation should be successful, so there is no point in dividing it into different tests. I renamed the name of the test